### ISSUE TITLE
Improve bundle configure.pl scripts to emit compiler commands into log

### ISSUE THREAD

**Issue by jengelh** [polymake/polymake#13]
I have a case of [this] that I am chasing. While I can pull the source it tried to compile sorta from configure.pl, it would be  more helpful if the log contained the compiler command, because _that's_ not obvious from bundled/libnormaliz/support/configure.pl itself.

```
---- libnormaliz ----

Could not compile a test program checking for libnormaliz.
The most probable reasons are that the library is installed at a non-standard location,
is not configured to build a shared module, or missing at all.
Also make sure that libnormaliz was built with the same C++ library as polymake,especially if the errors below show missing symbols cont
aining std::__1::vector or std::vector.The complete error log follows:

/usr/lib64/gcc/x86_64-suse-linux/12/../../../../x86_64-suse-linux/bin/ld: /usr/lib64/gcc/x86_64-suse-linux/12/../../../../lib64/libnorma
liz.so: undefined reference to symbol '_ZN6eantic15renf_elem_classD1Ev'
/usr/lib64/gcc/x86_64-suse-linux/12/../../../../x86_64-suse-linux/bin/ld: /usr/lib64/libeanticxx.so.1: error adding symbols: DSO missing
 from command line
collect2: error: ld returned 1 exit status

Please install the library and specify its location using --with-libnormaliz option, if needed.
```

**Comment by benlorenz** [comment]
Hi,
thanks for the report, a preliminary patch to add the command line for failing configure checks is available here: https://polymake.org/downloads/configure_compile_commands.patch
I will check if we can include this in the next release.
Alternatively you could also add `-v` to `CXXFLAGS` to get the command line from the verbose gcc output.

Regarding the normaliz configuration: Unfortunately normaliz doesn't provide any pkg-config file or something similar to get the correct linker flags and our code to deduce them from the configuration is somewhat outdated / incomplete. This patch should improve things: https://polymake.org/downloads/polymake_normaliz_libs.patch
This should work with all reasonably new eantic installations. I might need to adapt it slightly to be able to deal with eantic versions that don't have `libeanticxx`, if these are still relevant, before including this for our next release.

Best,
Benjamin

**Comment by benlorenz** [comment]
Both changes are included in polymake 4.9.
